chpl/chpl-service/generateDailySEDDetailsDownload.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+(set -o igncr) 2>/dev/null && set -o igncr; # this comment is required to trick cygwin into dealing with windows vs. linux EOL characters
+
+# to generate the XML file regularly, add a line to a crontab on the machine hosting the application that looks something like:
+# 15 5 * * * cd /some/directory/chpl-api/chpl/chpl-service && ./generateDailySEDDetailsDownload.sh
+# This will run it at 0515 UTC, which (depending on DST) is 0015 EST
+
+# create timestamp and filename
+TIMESTAMP=$(date "+%Y.%m.%d-%H.%M.%S")
+log=logs/log.generateSEDDetailsDownload.$TIMESTAMP.txt
+
+# deal with spaces in filenames by saving off the default file separator (including spaces)
+# and using a different one for this application
+SAVEIFS=$IFS
+IFS=$(echo -en "\n\b")
```

Comment:
Unless you're explicitly creating files with spaces in them, this part isn't required, and just serves to confuse the reader